include/swift/SIL/InstWrappers.h

```diff
@@ -69,6 +69,8 @@ struct LoadOperation {
 };
 
 /// A wrapper type for writing generic code against conversion instructions.
+///
+/// Forwards a single operand in first operand position to a single result. 
```

Comment:
Is it "in first operand position" in the case of `tuple_pack_extract %index of %tuple`?

Reply:
ConversionOperation hard-codes first operand:

`SILValue getConverted() { return inst->getOperand(0); }`

TuplePackExtract is not a conversion. Mainly because it doesn't forward all incoming references. Although that's less relevant here because it only supports guaranteed values.

Reply:
Oops, I missed that this wasn't a comment on `ForwardingOperation`.